### ISSUE TITLE
Fix Test: Authentication.tsx

### ISSUE THREAD

**Issue by leesamuel423** [s-hc/LOLDoku#21]
🌳 BRANCHING:
- Branch off [main]
- PR into [main]

😿 ISSUE:
- Currently, the .env variable for the `VITE_BACKEND_URL` is used with `import.meta.env.VITE_BACKEND_URL` to route to backend `/auth/google` in `client/components/Authentication.tsx`
- This solution currently works, and authentication works properly with no issues. However, the unit tests no longer work because of the following error:

```
/LOLDoku/src/client/components/Authentication.tsx:18
        var backendUrl = "".concat(import.meta.env.VITE_BACKEND_URL, "/auth/google");
                                          ^^^^

    SyntaxError: Cannot use 'import.meta' outside a module
```

This issue triggers test faillure in Navbar.test.tsx and Authentication.test.tsx

🟢 TASK:
- Modify settings to allow for proper reading of environment variable.
- Maybe utilization of [babel-plugin-transformer](https://github.com/javiertury/babel-plugin-transform-import-meta/)
- You will need to `npm run build` and then run `npm run test` after making changes
- Please ensure that before making the PR, manual run through authentication portion of application via `npm run dev`

**Comment by leesamuel423** [comment]
Resolved in PR #24

**Comment by leesamuel423** [comment]
Resolved in PR #24
